descqa/StellarMassFunction.py

```diff
@@ -0,0 +1,353 @@
+from __future__ import print_function, division, unicode_literals, absolute_import
+import os
+import math
```

Comment:
remove this line because `math` is not used.

Reply:
Done.